### ISSUE TITLE
typescript error with sequelize literal

### ISSUE THREAD

**Issue by fareshan** [sequelize/sequelize#11080]
This does not work:
```ts
const aas = await A.findAll({
      where: Sequelize.literal(
        `id IN (
            SELECT "aaId"
            FROM "Pp"
            WHERE "ppId" = 10
            )
        `,
      ),
    });
```

The typescript error:
```bash
Argument of type '{ where: Literal; }' is not assignable to parameter of type 'FindOptions'.
  Types of property 'where' are incompatible.
    Type 'Literal' is not assignable to type 'WhereAttributeHash | AndOperator | OrOperator | Where | undefined'.
      Type 'Literal' is not assignable to type 'WhereAttributeHash'.
        Index signature is missing in type 'Literal'.ts(2345)
```

## Environment
**sequelize-typescript**: 1.0.0-beta.3
Dialect:
- [ ] mysql
- [ X ] postgres
- [ ] sqlite
- [ ] mssql
- [ ] any
Dialect **library** version: pg version 6.4.2
Database version: 9.6.12
Sequelize version: 5.8.10
Node Version: 8.11.4
OS: macOS 10.14.3
TypeScript version: 3.4.3 (or 3.3.3 same issue)
Tested with latest release:
- [ ] No
- [ X ] Yes, specify that version: 5.8.10

**Comment by leejh3224** [comment]
refer to [this issue](https://github.com/sequelize/sequelize/issues/9410)

**Comment by fareshan** [comment]
@leejh3224 this looks more like a turnaround than a solution

**Comment by fareshan** [comment]
this issue is fixed with this fix:
[https://github.com/sequelize/sequelize/pull/10990](url)

**Comment by papb** [comment]
Fixed by #10990 

(the link by @fareshan [above](https://github.com/sequelize/sequelize/issues/11080#issuecomment-510691072) was broken)
